src/ftrackWidget.ts

```diff
@@ -186,11 +235,10 @@ function onDocumentKeyDown(event) {
     "keyCode",
     "which",
   ];
-  const eventData = {};
-  for (let i = 0; i < fields.length; i += 1) {
-    const field = fields[i];
-    eventData[field] = event[field];
-  }
+
+  const eventData: Partial<KeyboardEvent> = Object.fromEntries(
```

Comment:
Doesn't looks like this get transpiled, so we are raising the requirement to browsers supporting [Object.fromEntries](https://caniuse.com/?search=fromEntries) and [arrow functions](https://caniuse.com/?search=arrow%20function).

Personally, I would have kept the for-loop, but I think the browser support is enough for this to be ok.